docs/source/docker/advanced/code_examples/python_run_active_learning.py

```diff
@@ -0,0 +1,74 @@
+import lightly
+
+# Create the Lightly client to connect to the API.
+client = lightly.api.ApiWorkflowClient(token="TOKEN", dataset_id="DATASET_ID")
+
+# Schedule the docker run with 
+#  - "active_learning.task_name" set to your task name
+#  - "method" set to "coral"
+# All other settings are default values and we show them so you can easily edit
+# the values according to your need.
+client.schedule_compute_worker_run(
+    worker_config={
+        "enable_corruptness_check": True,
+        "remove_exact_duplicates": True,
+        "enable_training": False,
+        "pretagging": False,
+        "pretagging_debug": False,
+        "method": "coral",
+        "stopping_condition": {
+          "n_samples": 0.1,
+          "min_distance": -1
+        },
+        "scorer": "object-frequency",
+        "scorer_config": {
+          "frequency_penalty": 0.25,
+          "min_score": 0.9
+        },
+        "active_learning": {
+          "task_name": "my-classification-task",
+          "score_name": "uncertainty_margin"
```

Comment:
Do we already have follow up issues to make sure we don't have score_name, scorer twice in the config?

Reply:
No, not yet but we have one that's related. I will update this one.